Traceback (most recent call last):
  File "main.py", line 218, in <module>
    with open('args_parser/commandline_args_test' + args.exp_name + '.txt', 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'args_parser/commandline_args_testCNN_mnist_custom_BK_2layer_bk1_20_loss_distance_1.txt'
